openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -333,7 +333,8 @@ This endpoint is used by a Credential Issuer that is already interacting with an
 
 ## Credential Offer {#credential-offer}
 
-The Credential Issuer sends Credential Offer using an HTTP GET request or an HTTP redirect to the Wallet's Credential Offer Endpoint defined in (#client-metadata).
+The Credential Issuer makes a Credential Offer by directing the End-User's
+user-agent to the Wallet's Credential Offer Endpoint defined in (#client-metadata). The Credential Issuer MAY render a QR code containing the Credential Offer that can be scanned by the End-User using a Wallet, or a link that the End-User can click.
```

Comment:
I am confused. when the wallet scans the credential offer QR code, user agent/browser is not opened.

```suggestion
The Credential Issuer sends a Credential Offer to the Wallet's Credential Offer Endpoint defined in (#client-metadata.
Some of the ways the Credential Issuer can do so are by rendering a QR code containing the Credential Offer that can be scanned by the End-User using a Wallet, or providing a URL containing the Credential Offer that the End-User can click and open the wallet.
```